src/migrations/20240917071436-add-foreign-key-to-role-permissions.js

```diff
@@ -0,0 +1,20 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        UPDATE role_permission SET environment = null where environment = '';
+        DELETE FROM role_permission WHERE environment IS NOT NULL AND environment NOT IN (SELECT name FROM environments);
```

Comment:
Delete all permissions from role_permission where environment is not null and does not exist in the environments table